### ISSUE TITLE
Don't show pinned tabs in top sites

### ISSUE THREAD

**Issue by Verdi** [mozilla/activity-stream#729]
If I already have a tab pinned, don't show it in my top sites. I don't need to open a copy of it.

**Comment by rlr** [comment]
nice. Should we go further and apply that to any url that is in an open tab?

**Comment by Verdi** [comment]
Yeah maybe. I almost said that. I can see the case where you want to open a new tab of the same site though, like maybe Wikipedia or YouTube. I guess I was thinking if you have a site pinned that's a pretty good signal you don't need another copy.

**Comment by bryanbell** [comment]
It's valid to say that a top site with a corresponding pinned tab feels redundant, but unfortunately pinned tabs don't show on every window, so sometimes an important site won't be represented twice.  Also we do provide a handy "dismiss" option that will remove a top site if the user feels that it's not helpful to them.

**Comment by Verdi** [comment]
I think my argument works whether it's in the same window or a new window. Let's say I have gmail, twitter and facebook all pinned in a window. When I open another window it seems like there is a decent chance I don't want to open another copy of those pages - that's why I pinned them. 

Sure I can dismiss a site but that's work I don't want to do. It would be nice if activity stream did the work for me and showed me 3 new top sites in place of the ones I have pinned.
